pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -680,6 +680,18 @@ type PoolSpec struct {
 	// +optional
 	// +nullable
 	Quotas QuotaSpec `json:"quotas,omitempty"`
+
+	// The pg settings
+	// +optional
+	// +nullable
+	PGs int `json:"pgs,omitempty"`
+
+	// The autoscale mode settings
+	// +kubebuilder:default:=on
```

Comment:
Default values in the crds actually cause issues. We had to revert one recently in #11951. So let's instead set the default in code rather than in the crd.
```suggestion
```